src/d3a/d3a_core/simulation.py

```diff
@@ -88,21 +88,21 @@ def __init__(self, setup_module_name: str, simulation_config: SimulationConfig =
         self.run_start = None
         self.paused_time = None
 
+        # change_global_config(**simulation_config.__dict__)
         self._load_setup_module()
         self._init(**self.initial_params)
         self._init_events()
 
     def _set_traversal_length(self):
-        if ConstSettings.GeneralSettings.MAX_OFFER_TRAVERSAL_LENGTH is None:
-            no_of_levels = self._get_setup_levels(self.area) + 1
-            num_ticks_to_propagate = no_of_levels * 2
-            ConstSettings.GeneralSettings.MAX_OFFER_TRAVERSAL_LENGTH = int(num_ticks_to_propagate)
-            time_to_propagate_minutes = num_ticks_to_propagate * \
-                self.simulation_config.tick_length.seconds / 60.
-            log.error("Setup has {} levels, offers/bids need at least {} minutes "
-                      "({} ticks) to propagate.".format(no_of_levels, time_to_propagate_minutes,
-                                                        ConstSettings.GeneralSettings.
-                                                        MAX_OFFER_TRAVERSAL_LENGTH,))
+        no_of_levels = self._get_setup_levels(self.area) + 1
+        num_ticks_to_propagate = no_of_levels * 2
```

Comment:
This is not in the context of this story, butI was thinking how could we remove this "2" magic value from here, and one way is to change the _get_setup_levels method to actually access the "min_offer_age" IAA parameter. That way we can calculate the exact amount of ticks required to propagate an offer without relying on having always min_offer_age = 1. Maybe we can open a small task for handling this

Reply:
Yes, agreed. Bu I suggest to open a small task and deal with it separately. Will do so.

Reply:
Cool, thanks!